reladomo/src/main/java/com/gs/fw/common/mithra/attribute/NonPrimitiveAttribute.java

```diff
@@ -317,4 +317,6 @@ public Operation notIn(Set<Type> set)
 //    public abstract Operation notIn(Set<Type> set);
 
     public abstract void setSqlParameter(int index, PreparedStatement ps, Object o, TimeZone databaseTimeZone, DatabaseType databaseType) throws SQLException;
+
+    public abstract String formattedValue(Object object);

```

Comment:
I could pull up a default implementation which returns String.valueOf() but I thought it was a little safer not to.

Reply:
The worse thing that'll happen is an extra quote :smile:. I think I prefer that to the code duplication.

Reply:
Pulled up the implementation that quotes strings